troubleshooting failed domain transfers. updating instructions (#14459)
cloudflare/cloudflare-docs: content/registrar/troubleshooting.md

## Restart your transfer

{{<Aside type="note">}}

> This does not apply to .uk domains.

Reviewer: ```suggestion
This solution does not apply to `.uk` domains.
```

{{</Aside>}}

1. Log in to [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
2. Select **Domain Registration** > **Manage Domains**.

> 3. On the right of the domain name, select **Manage**.

Reviewer: ```suggestion
3. Find the correct domain and select **Manage**.
```

Reviewer: We should avoid writing out locations like this, in case the dash changes things around

Author: Agree!
